File: demo/src/main/java/uk/co/threefi/dataload/structure/RawRow.java
package uk.co.threefi.dataload.structure;

import java.io.Serializable;

public class RawRow implements Serializable {
  private static final long serialVersionUID = -812004521983071103L;

  private String[] rawVals = null;

  public void setRawVals(String[] rawVals) {
    this.rawVals = rawVals;
  }

  public String getColumnVal(int columnIndex) {
    if(rawVals != null  && columnIndex < rawVals.length) {
      return rawVals[columnIndex];
    }
    return "";
  }

}

File: demo/src/main/scala/kaspar/dataload/structure/RawRow.scala
package kaspar.dataload.structure

class RawRow extends Serializable {

  private var rawVals:Seq[String] = null

  def setRawVals(rawVals: Seq[String]): Unit = {
    this.rawVals = rawVals
  }

  def getColumnVal(columnIndex: Int): String = {
    if (rawVals != null && columnIndex < rawVals.length) return rawVals(columnIndex)
    ""
  }

}
